tools/StaticAnalysis/SerializedCmdlets/Microsoft.Azure.Commands.Compute.dll.json

```diff
@@ -30873,7 +30873,7 @@
           "ValidateNotNullOrEmpty": true
```

Comment:
Please remove this change, and please add a description of the change (the new parameter) to the Changelog.md for your module, here: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Compute/ChangeLog.md